cypress/integration/pages/mediaAssetPage/helpers.js

```diff
@@ -0,0 +1,16 @@
+export const getBlockByType = (blocks, blockType) => {
+  let blockData;
+
+  blocks.forEach(block => {
+    if (!blockData && block.type === blockType) {
+      blockData = block;
+    }
+  });
+  return blockData;
```

Comment:
You can do this type of searching a bit easier with `.find`

``` let blockData = blocks.find(block => block.type === blockType)```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
I tried it but couldn't get it to work :0